posts/2021-09-14-open-liberty-stack-gradle.adoc

```diff
@@ -31,14 +31,14 @@ To begin, you need three things:
 
 - Kubernetes cluster (i.e. Openshift, Minikube). Be sure to log in. Odo will interact with your kubernetes cluster.
 - Openshift Do. If you have not already installed odo, do so now by following the instructions outlined in the link:https://odo.dev[odo documentation].
-- An application using Gradle. You can use your own application, or use one of the applications provided by the stack, such as the link:https://github.com/OpenLiberty/application-stack-intro[intro] or link:https://github.com/OpenLiberty/application-stack-starters[starter] application. For this blog, we will use the intro application. This is a basic application that uses JAX-RS and MicroProfile technologies. For instructions on how to use the starter application, see link:https://github.com/OpenLiberty/application-stack/wiki/Using-the-Default-Starter#getting-started[[Using the Default Starter: Getting started]].
+- An application using Gradle. You can use your own application, or use one of the applications provided by the stack, such as the link:https://github.com/OpenLiberty/devfile-stack-intro[intro] or link:https://github.com/OpenLiberty/devfile-stack-starters[starter] application. For this blog, we will use the intro application. This is a basic application that uses JAX-RS and MicroProfile technologies. For instructions on how to use the starter application, see link:https://github.com/OpenLiberty/devfile-stack/wiki/Using-the-Default-Starter#getting-started[[Using the Default Starter: Getting started]].
```

Comment:
sorry I missed this before, but as long as we're here, the link is double-bracketted